ffmpeg.c

```diff
@@ -295,7 +295,7 @@ static int ffmpeg_get_oformat(struct ffmpeg *ffmpeg){
     if (ffmpeg->tlapse == TIMELAPSE_APPEND){
         ffmpeg->oc->oformat = av_guess_format ("mpeg2video", NULL, NULL);
         if (ffmpeg->oc->oformat) ffmpeg->oc->oformat->video_codec = MY_CODEC_ID_MPEG2VIDEO;
-        strncat(ffmpeg->filename, ".mpg", 4);
+        strncat(ffmpeg->filename, ".mpg", 5);
```

Comment:
Since the source string is a string literal, these changes can just be a `strcat` rather than `strncat`. That will prevent this issue in the future if the extensions are ever changed, since you won't have to keep track of the string size.

Reply:
You're right. Will change to use `strcat`.